tests/manage/rgw/test_host_node_failure.py

```diff
@@ -72,6 +109,37 @@ def test_rgw_host_node_failure(
         if noobaa_pod_node is None:
             assert False, "Could not find the NooBaa DB pod"
 
+        # Validate if RGW pod and noobaa-db are hosted on same node else skip test
+        log.info("Validate if RGW pod and noobaa-db are hosted on same node")
+        rgw_pod_obj = get_rgw_pods()
+        rgw_pod_node_list = [
+            rgw_pod.get().get("spec").get("nodeName") for rgw_pod in rgw_pod_obj
+        ]
+        if not list(set(rgw_pod_node_list).intersection(noobaa_pod_node.name.split())):
+            log.info(
+                "Unschedule other two nodes such that RGW "
+                "pod moves to node where NooBaa DB pod hosted"
+            )
+            worker_node_list = get_worker_nodes()
+            node_names = list(set(worker_node_list) - set(noobaa_pod_node.name.split()))
+            unschedule_nodes(node_names=node_names)
+            ocp_obj = OCP(kind=constants.POD, namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+            rgw_pod_obj[0].delete()
```

Comment:
we are already doing the drain, do we need to still delete the rgw pod?

Reply:
Here, making nodes has `SchedulingDisabled` not drain so new pods doesn't come up on that node but doesn't move existing pods on the node.